app/src/main/java/org/gdg/frisbee/android/api/PlusApi.java

```diff
@@ -10,4 +14,15 @@
 public interface PlusApi {
     @GET("people/{gplusId}?fields=image&key=" + BuildConfig.IP_SIMPLE_API_ACCESS_KEY)
     Call<ImageInfo> getImageInfo(@Path("gplusId") String gplusId);
+
+    @GET("people/{gplusId}?fields=image,aboutMe,tagline,urls,url,cover,displayName&key="
+        + BuildConfig.IP_SIMPLE_API_ACCESS_KEY)
+    @Nullable Person getPerson(@Path("gplusId") String gplusId);
+
+    @GET("people/{gplusId}/activities/public?fields="
+        + "nextPageToken,"
+        + "items(id,published,url,object/content,verb,"
+        + "object/attachments,annotation,object(plusoners,replies,resharers))"
+        + "&key=" + BuildConfig.IP_SIMPLE_API_ACCESS_KEY)
```

Comment:
Using key here (instead of in an interceptor) makes it more explicit where the authentication takes place. I prefer this solution of a

```
public class ApiKeyAdder implements Interceptor {
    @Override
    public Response intercept(Chain chain) throws IOException {
        Request original = chain.request();

        HttpUrl newUrl = original.url().newBuilder()
            .addQueryParameter("key", BuildConfig.IP_SIMPLE_API_ACCESS_KEY).build();
        Request.Builder requestBuilder = original.newBuilder().url(newUrl);

        Request request = requestBuilder.build();
        return chain.proceed(request);
    }
}
```